components/blog/BlogCard.vue

```diff
@@ -7,7 +7,7 @@
       :title="article.coverImgAlt"
     />
     <div class="d-flex flex-column align-items-start">
-      <span class="blog__date mt-3">{{ formatDate(article.createdAt) }}</span>
+      <span class="blog__date mt-3">{{ formatDate(article.publishedAt) }}</span>
```

Comment:
Published date will be the same that it used to be right? It will not change the date after migration I hope?

Reply:
Yes Yes it is same as before..